code/game/objects/items/granters.dm

```diff
@@ -392,3 +392,21 @@
 		icon_state = "blankscroll"
 
 // I did not include mushpunch's grant, it is not a book and the item does it just fine.
+
+/obj/item/book/granter/martial/krav_maga
+	martial = /datum/martial_art/krav_maga
```

Comment:
I don't think this needs to be changed due to the effort requirement in doing so, but coming up with another martial art to replace this since we already have the krav magloves is an idea for someone in the future.